apps/etterlatte-brev-api/src/test/kotlin/no/nav/etterlatte/brev/behandling/SakOgBehandlingServiceTest.kt

```diff
@@ -0,0 +1,205 @@
+        assertEquals(2000, avansertBeregning.utbetalingsinfo?.beloep)
+
+        coVerify(exactly = 1) { vedtaksvurderingKlient.hentVedtak(BEHANDLING_ID, ACCESS_TOKEN) }
+        coVerify(exactly = 1) { vedtaksvurderingKlient.hentVedtak(AVANSERT_BEHANDLING_ID, ACCESS_TOKEN) }
+        coVerify(exactly = 2) { grunnlagKlient.hentGrunnlag(SAK_ID, ACCESS_TOKEN) }
+        coVerify(exactly = 1) { beregningKlient.hentBeregning(BEHANDLING_ID, ACCESS_TOKEN) }
+        coVerify(exactly = 1) { beregningKlient.hentBeregning(AVANSERT_BEHANDLING_ID, ACCESS_TOKEN) }
+    }
+
+    private fun opprettEnkelBeregning() = mockk<BeregningDTO> {
+        every { beregningsperioder } returns listOf(
+            opprettBeregningsperiode(
+                YearMonth.now(),
+                beloep = 3063
+            )
+        )
+    }
+
+    private fun opprettAvansertBeregning() = mockk<BeregningDTO> {
+        every { beregningsperioder } returns listOf(
+            opprettBeregningsperiode(
+                YearMonth.of(2022, 6),
+                YearMonth.of(2022, 12),
+                1000
+            ),
+            opprettBeregningsperiode(
+                YearMonth.of(2023, 1),
+                YearMonth.of(2023, 6),
+                2000
+            ),
+            opprettBeregningsperiode(
+                YearMonth.of(2023, 7),
+                beloep = 3000
+            )
+        )
+    }
+
+    private fun opprettVedtak() = mockk<Vedtak> {
+        every { sak.ident } returns "ident"
+        every { vedtakId } returns 123L
+        every { type } returns VedtakType.INNVILGELSE
+        every { virk } returns Periode(YearMonth.now(), null)
+        every { vedtakFattet } returns VedtakFattet(SAKSBEHANDLER_IDENT, "Ansvarlig enhet", ZonedDateTime.now())
+        every { attestasjon } returns Attestasjon(ATTESTANT_IDENT, "Attestant enhet", ZonedDateTime.now())
+    }
+
+    private fun opprettGrunnlag() = mockk<Grunnlag> {
+        every { sak } returns mapOf(
+            Opplysningstype.SPRAAK to opprettOpplysning(Spraak.NB.toJsonNode()),
+            Opplysningstype.INNSENDER_SOEKNAD_V1 to opprettOpplysning(
+                InnsenderSoeknad(
+                    PersonType.INNSENDER,
+                    "Innsend",
+                    "Innsender",
+                    FNR
+                ).toJsonNode()
+            )
+        )
+        every { metadata.sakId } returns 123L
+        every { soeker } returns mapOf(
+            Opplysningstype.NAVN to opprettOpplysning(Navn("Soek", "Soekeren").toJsonNode()),
+            Opplysningstype.FOEDSELSNUMMER to opprettOpplysning(FNR.toJsonNode())
+        )
+        every { familie } returns listOf(mapOf())
+        every { hentAvdoed() } returns mapOf(
+            Opplysningstype.NAVN to opprettOpplysning(Navn("Avdoed", "Avdoederen").toJsonNode()),
+            Opplysningstype.DOEDSDATO to opprettOpplysning(LocalDate.now().toJsonNode())
+        )
+    }
```

Comment:
Dersom det ikke er en spesiell grunn for at det _ikke_ er brukt, så er det fint å bruke test-liben vi har opprettet for Grunnlag. Sjekk ut `GrunnlagTestData` og bruken av det

Reply:
Tatt i bruk nå 👌